data/json/recipes/food/frozen.json

```diff
@@ -7,7 +7,7 @@
     "skill_used": "cooking",
     "difficulty": 1,
     "time": "10 m",
-    "autolearn": true,
+    "book_learn": [ [ "dairy_book", 1 ] ],
```

Comment:
I think this is okay to be autolearned

Reply:
I'm good to let it be autolearned but for some reason all the milkshake items when autolearned were failing travis C++ checks and I could only fix it by making them book learned.  There was probably something off key that I couldn't find with spellcheck, linting or anything else.  hopefully when I commit these back to autolearned they won't fail checks.